source/slang/slang-capabilities.capdef

```diff
@@ -691,3 +722,38 @@ alias all = _sm_6_7 + hlsl_nvapi
           | spirv_1_5 + sm_6_7
              + ser + shaderclock + texturefootprint + fragmentshaderinterlock + spvGroupNonUniformPartitionedNV
              + spvRayTracingMotionBlurNV + spvRayTracingMotionBlurNV;
+
+/// Test Capabilities
+
+alias TEST_ADD_1 = _sm_4_1 | _GLSL_130 | spirv_1_1 | metal
```

Comment:
Not a priority right now, but we need a way to make sure these doesn't get into release builds.